experimental/oak_baremetal_channel/README.md

```diff
@@ -0,0 +1,4 @@
+# Oak Baremetal Channel
+
+This is a transport-agnostic communication channel between the guest VM and the
+host. Here is the [SPEC](SPEC.MD).
```

Comment:
```suggestion
host. Here is the [specification](SPEC.MD).
```

Or is `SPEC` an abbreviation with a specific meaning here?

Reply:
Not sure, it was pre-existing, so I'd leave it for now.